### DISCUSSION TITLE
Can I use form instance inside filterDropdown?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#42478] [Q&A]
Now I'm using raw inputs with different state, it's possible to use also form instance and get **ontable** **change** inputs values??

![Screenshot 2023-05-18 alle 15 21 21](https://github.com/ant-design/ant-design/assets/40239902/97f79efa-a4cd-4169-880e-d888927f415e)

**Comment by vitalijalbu** [comment]
https://github.com/ant-design/ant-design/issues/42479

**Comment by BoyYangzai** [comment]
@vitalijalbu 
Sorry, I don't really understand what you're wanting, but do you want to re-place the filtered results in the table after you get the dynamic filtered results from the form query? Why not use `useState` to make your data a `State`?

**Reply by vitalijalbu** [reply]
Hi, try to see the codesandbox. I'd like to set a global state from form component, on each column... but seems impossible.
